src/grid_interpolator.cc

```diff
@@ -0,0 +1,99 @@
+#include "flyft/grid_interpolator.h"
+
+#include <fstream> 
+#include <cassert>
+
+namespace flyft
+{
+GridInterpolator::GridInterpolator():
+    n_(0,0,0)
+    {
+    }
+    
+GridInterpolator::GridInterpolator(const std::string& s)
+    {
+    int nx, ny, nz;
+    double dx, dy, dz;
+    
+    std::ifstream indata(s);
+    if(!indata)
+        {
+        throw std::runtime_error("File failed to open");
+        }
+        
+    indata>>nx>>ny>>nz>>dx>>dy>>dz;
```

Comment:
Use whitespace to make this more readable, and read directly into the member variables when it's possible.
```suggestion
    int nx, ny, nz;
    indata >> nx >> ny >> nz >> dx_ >> dy_ >> dz_;
```